src/backends/policy_engine/souffle/check_policy_compliance_test.sh

```diff
@@ -22,19 +22,20 @@ FAIL_RESULT="fail"
 function printUsageAndExit() {
   echo "Usage: "
   echo "  check_policy_compliance_test.sh \ "
-  echo "     <cmd> ${PASS_RESULT}|${FAIL_RESULT} <ir_file> <sql_policy_rules_file> <policy_engine>"
+  echo "     <cmd> ${PASS_RESULT}|${FAIL_RESULT} <ir|proto> <file> <sql_policy_rules_file> <policy_engine>"
   exit 1
 }
 
-if [ $# -ne 5 || $# -ne 4]; then
+if [ $# -ne 6 || $# -ne 5]; then
```

Comment:
What do `6` and `5` mean here?

Reply:
min # args and max # args, I added vars for them